### ISSUE TITLE
The fmodel can't transport the running_mean and running_var of BN layer to the original model

### ISSUE THREAD

**Issue by ensiwalk** [facebookresearch/higher#103]
The fmodel can't transport the running_mean and running_var to the original model, which may cause that the loss explosion when the model in meta_test dataset, please fix it or give me some advice on how to address it  gracefully? Thanks a lot for that!

**The sample code and related results are as follows：**

   
        for i_task in range(args.n_batch):
            
            x_spti, x_qryi = torch.from_numpy(x_spt[i_task]).float().to(device), torch.from_numpy(x_qry[i_task]).float().to(device)

            fmodel = higher.patch.monkeypatch(model, device=device, copy_initial_weights=False, track_higher_grads=True)

            support_preds = fmodel(x_spti)
            support_loss = loss(support_preds, x_spti)
            grad = torch.autograd.grad(support_loss, fmodel.fast_params, create_graph=False,retain_graph=True,allow_unused=True)
            fast_weights = list(map(lambda p: p[1] -adapt_lr * p[0], zip(grad, fmodel.fast_params)))

            for step in range(1,args.adapt_steps):
                fmodel.fast_params = fast_weights
                support_preds = fmodel(x_spti)
                support_loss = loss(support_preds, x_spti)

                grad = torch.autograd.grad(support_loss, fmodel.fast_params, create_graph=False,retain_graph=True,allow_unused=True)

                fast_weights = list(map(lambda p: p[1] - adapt_lr * p[0], zip(grad, fmodel.fast_params)))
            fmodel.fast_params = fast_weights
            x_hat_qry = fmodel(x_qryi)
            iter_error_temp = loss(x_hat_qry,x_qryi)
            iter_error = iter_error + iter_error_temp
        iter_error = iter_error/args.n_batch
        iter_error.backward()
        optimizer.step()
`
**1  related result** 
>In [1] fmodel.state_dict()['encoder.base1.conv1.bn.running_mean'] 
>Out[1] tensor([ 0.0519, -0.1342, -0.1098, -0.0754, -0.0366, -0.0326, -0.0846, -0.0088,
         0.0524, -0.0084, -0.0287,  0.0098,  0.0276,  0.0137, -0.0608,  0.0782,
        -0.0773,  0.1453, -0.0556,  0.0077, -0.0784,  0.1525, -0.0749, -0.1261,
        -0.0518,  0.0650,  0.0663, -0.0138, -0.0267,  0.0617, -0.0196,  0.0009],
       device='cuda:0')
>In [2]  model.state_dict()['encoder.base1.conv1.bn.running_mean'] 
>Out [2]  tensor([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
        0., 0., 0., 0., 0., 0., 0., 0.], device='cuda:0')

**the final results comparison between with and without running_mean and running_var**
![func2](https://user-images.githubusercontent.com/31041062/110769048-bc3ecb00-8292-11eb-8767-9696ad0dda3d.png)

The orange line means the model trianed assisted by higher Lib, and the blue line means the model trained assisted by another Lib. Since using the same random seed, the both models' parameters are all the same except running_mean and running_var. It can be seen that at the very beginning, there was an obvious loss explosion. I don't know how to address it gracefully besides pass parameters layer by layer manually.
